src/components/layout/TopNavigation.tsx

```diff
@@ -96,6 +96,9 @@ const TopNavigation: React.FC<TopNavigationProps> = ({ title }) => (
       <MenuItem>
         <Link to="/submit-a-talk">Ajukan Topik</Link>
       </MenuItem>
+      <MenuItem>
+        <Link to="/faq">F.A.Q</Link>
```

Comment:
```suggestion
        <Link to="/faq">FAQ</Link>
```

Untuk singkatan tidak usah diberi titik.